app/src/main/java/org/jamienicol/episodes/EpisodesListFragment.java

```diff
@@ -194,6 +194,14 @@ public void bindView(View view, Context context, Cursor cursor) {
 					DateFormat.getDateInstance().format(date);
 				dateView.setText(dateText);
 				dateView.setVisibility(View.VISIBLE);
+
+				// grey out episode name if upcoming episode
+				if (date.after(new Date())) {
+					nameView.setTextColor(context.getResources().getColor(android.R.color.secondary_text_dark));
```

Comment:
Could you also please set the text colour of the date to be the same as the episode name. It looks a bit strange when they're different colours. Thanks!